massa-protocol-exports/src/tests/mock_network_controller.rs

```diff
@@ -96,10 +96,12 @@ impl MockNetworkController {
     /// send operations
     /// todo inconsistency with names
     pub async fn send_operations(&mut self, source_node_id: NodeId, operations: Operations) {
+        let serialized = operations.iter().map(|_| Default::default()).collect();
```

Comment:
Why we don't store real serialize operations?

Reply:
Because the serialized form is unrelated to the protocol tests. You can see it as a kind of mock.